<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReactionsManager</class>
 <widget class="QWidget" name="ReactionsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_reactions">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="listWidget_reactions"/>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Reaction parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="2" column="2">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Performer alias</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Performer type</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="cmbx_reverse_performer_behavior_type"/>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="chbx_reversible_flag">
            <property name="text">
             <string>Reversible</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_reverse_behavior">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Reverse behavior</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cmbx_reaction_performer_type"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Performer behavior</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Alias</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cmbx_reaction_performer_behavior_type"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Trigger</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QComboBox" name="cmbx_reaction_event"/>
          </item>
          <item row="7" column="2" colspan="2">
           <widget class="QComboBox" name="cmbx_reverse_performer_behavior_preset"/>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="le_reaction_alias"/>
          </item>
          <item row="11" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QComboBox" name="cmbx_reaction_performer_behavior_preset"/>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="cmbx_reaction_performer_alias">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="8" column="0" rowspan="3" colspan="4">
           <widget class="QGroupBox" name="groupBox_valid_states">
            <property name="title">
             <string>Valid states</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="chbx_state_configuring">
               <property name="text">
                <string>Configuring</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbx_state_ad">
               <property name="text">
                <string>Disarming</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbx_state_partial_armed">
               <property name="text">
                <string>Partial Armed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbx_state_armed">
               <property name="text">
                <string>Armed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbx_state_da">
               <property name="text">
                <string>Arming</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbx_state_forced_disarmed">
               <property name="text">
                <string>Force Disarmed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chbx_state_disarmed">
               <property name="text">
                <string>Disarmed</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_add_apply">
           <property name="text">
            <string>Add reaction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_del_cancel">
           <property name="text">
            <string>Delete reaction</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget_reactions</tabstop>
  <tabstop>le_reaction_alias</tabstop>
  <tabstop>cmbx_reaction_event</tabstop>
  <tabstop>cmbx_reaction_performer_type</tabstop>
  <tabstop>cmbx_reaction_performer_alias</tabstop>
  <tabstop>cmbx_reaction_performer_behavior_type</tabstop>
  <tabstop>cmbx_reaction_performer_behavior_preset</tabstop>
  <tabstop>chbx_reversible_flag</tabstop>
  <tabstop>cmbx_reverse_performer_behavior_type</tabstop>
  <tabstop>cmbx_reverse_performer_behavior_preset</tabstop>
  <tabstop>chbx_state_configuring</tabstop>
  <tabstop>chbx_state_ad</tabstop>
  <tabstop>chbx_state_partial_armed</tabstop>
  <tabstop>chbx_state_armed</tabstop>
  <tabstop>chbx_state_da</tabstop>
  <tabstop>chbx_state_forced_disarmed</tabstop>
  <tabstop>chbx_state_disarmed</tabstop>
  <tabstop>btn_add_apply</tabstop>
  <tabstop>btn_del_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
